services/github/app.json

```diff
@@ -1,44 +0,0 @@
-{
```

Comment:
It's used by Heroku for the apps it runs for GitHub PRs.  We don't do that anymore, so it's not required.  I only noticed the file because I grepped for `GITHUB_OAUTH_TOKEN`.